### PR TITLE
Use VADS in Apply

### PR DESCRIPTION
**Pull request by mjlumetta**
https://github.com/department-of-veterans-affairs/vets-contrib/issues/3011

### CODE REVIEW

File: src/containers/ApplySuccess.tsx [hunk 1]
```diff
@@ -36,17 +34,27 @@ interface IOAuthCredentialsNoticeProps {
...
 // Mapping from the options on the form to Proper Names for APIs
-const apisToEnglishOauthList: any = {
+const apisToEnglishOauthList = {
   claims: 'Benefits Claims API',
   community_care: 'Community Care API',
   health: 'VA Health API',
   verification: 'Veteran Verfication API',
 };
 
+// this isn't great but we need to do it in the short term since Apply isn't based on the API definitions
```
Comment (kalilsn) [inline src/containers/ApplySuccess.tsx]: Can we just put this in its own module so we only define it once?

Comment (kalilsn) [reply src/containers/ApplySuccess.tsx]: and include the caveat in there that absolutely no other components should import it

Comment (dvogel) [reply src/containers/ApplySuccess.tsx]: Why would we define it more than once? I don't see another definition of it.

Comment (kalilsn) [reply src/containers/ApplySuccess.tsx]: 🤦‍♂ I thought this was the same as `formFieldsToFragments`. Nevermind!

Comment (kalilsn) [reply src/containers/ApplySuccess.tsx]: in my defense... the whole thing is confusing

File: src/containers/Apply.scss [hunk 2]
```diff
@@ -1,12 +0,0 @@
-@import '../styles/variables';
-
-.form-required-span {
-  color: $color-red;
-}
-
```
Comment (dvogel) [inline src/containers/Apply.scss]: Was this required class unused? I don't see it removed from anywhere in this PR.

Comment (mjlumetta) [reply src/containers/Apply.scss]: i don't remember if it was removed in a previous PR, but i couldn't find it anywhere else when i grep'd for it. so yeah, i think not used.

Comment (kalilsn) [reply src/containers/Apply.scss]: It's a duplicate of a class that's included in formation. I'm assuming we didn't have it back when we were just on uswds?

File: src/containers/ApplySuccess.tsx (hunk 1 above)
Comment (mjlumetta) [reply src/containers/ApplySuccess.tsx]: haha yeah, that is a good defense. either way it's horrible... but yeah, not sure that this ever worked in the first place?

**Pull request merged**